react.tsx

```diff
@@ -144,17 +144,18 @@ export function useFlags<F extends string=string, T extends string=string>(_flag
         }
     },[renderRef])
 
+    events.once('event', eventListener)
```

Comment:
This doesn't seem correct, it'd add an event listener per render??

Reply:
It will not since we are using the `events.once` method, which should guarantee we add it only once.